ui/redux/utils/reducerFactories.js

```diff
@@ -240,3 +240,46 @@ export const createObjectsByIdReducer = (updateActionType, initialState = {}, de
 
   return reducer
 }
+
+
+/**
+ * Factory function that creates a reducer for managing a state object that looks like:
+ *
+ * {
+ *    loading: false,
```

Comment:
I'd vote for renaming `loading` => `isLoading` and `error` => `errorMessage` so it's easier to guess the var type.